src/clj/weasel/repl/websocket.clj

```diff
@@ -39,7 +39,11 @@
 (defn repl-env
   "Returns a JS environment to pass to repl or piggieback"
   [& {:as opts}]
-  (let [compiler-env (env/default-compiler-env opts)
+  (let [ups-deps (cljsc/get-upstream-deps (java.lang.ClassLoader/getSystemClassLoader))
+        opts (assoc opts
+               :ups-libs (:libs ups-deps)
+               :ups-foreign-libs (:foreign-libs ups-deps))
+        compiler-env (env/default-compiler-env opts)
```

Comment:
@the-kenny not sure if plain `assoc` is the right thing here. Potentially users might want to influence what's in behind these keywords.